client/src/GameState.js

```diff
@@ -95,11 +95,9 @@ GameState.prototype.render = function(ctx) {
   if(states[0] && ready_for_render) {
     if(tick < states[0].tick){
       tick = states[0].tick; //global variables <3  ...
-      console.log("Local tick too long behind");
```

Comment:
I find these a bit useful to look at when there is some lag to understand what's happening.

But I guess we can reintroduce them when we're hunting latency bugs.